packages/datagateway-common/src/arrowtooltip.component.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect } from 'react';
+import Tooltip, { TooltipProps } from '@material-ui/core/Tooltip';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const arrowGenerator = (
```

Comment:
If this code has been migrated/copied over to `common` - can we remove the version in `dataview` and instead import it from `common` there? Otherwise we have code repetition 😬